numba/tests/test_withlifting.py

```diff
@@ -1213,5 +1220,30 @@ def foo():
         self.assertIn(msg, excstr)
 
 
+class TestMisc(TestCase):
+    # Tests for miscellaneous objmode issues. Run serially.
+
+    _numba_parallel_test_ = False
+
+    @linux_only
+    @needs_strace
+    def test_no_fork_in_compilation(self):
```

Comment:
Note to self. This test probably needs to be run as a subprocess test on the basis that previous execution of something which triggers a stateful call e.g. `uuid.uuid1.getnode()` is pretty likely.

Reply:
Addressed in d114552